neo/rawio/openephysrawio.py

```diff
@@ -485,11 +492,11 @@ def explore_folder(dirname):
             if (seg_index + 1) > info['nb_segment']:
                 info['nb_segment'] += 1
         elif filename.endswith('.spikes'):
-            s = filename.replace('.spikes', '').split('_')
-            if len(s) == 1:
-                seg_index = 0
+            s = re.findall("(_\d+)$", filename.replace('.spikes', ''))
```

Comment:
pep8 give a warning. I guess maybe you have to do \\ no ?
can you check ?

Reply:
@samuelgarcia Feel free commit the suggestion and see if it resolves the pep8 issue.